.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,27 @@
+name: New issue for Feature Suggestion
+about: Request our thoughts on your suggestion for a new feature for Homebrew.
+labels: features
+issue_body: false
+inputs:
+  - type: description
+    attributes:
+      value: Please note we will close your issue without comment if do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
```

Comment:
@MikeMcQuaid I think we're missing a `you` here (`if you do not...`).

Reply:
https://github.com/Homebrew/brew/pull/10474